### ISSUE TITLE
Path to font does not always work in SonataAdmin 4

### ISSUE THREAD

**Issue by jderusse** [sonata-project/SonataAdminBundle#7245]
### Environment

#### Sonata packages

<details><summary>show</summary>
<p>

```
sonata-project/admin-bundle              4.0.0-alpha-2 4.0.0-alpha-2 The missing Symfony Admin Generator
sonata-project/block-bundle              4.6.0         4.6.0         Symfony SonataBlockBundle
sonata-project/cache                     2.1.1         2.1.1         Cache library
sonata-project/datagrid-bundle           3.3.0         3.3.0         Symfony SonataDatagridBundle
sonata-project/doctrine-extensions       1.12.0        1.12.0        Doctrine2 behavioral extensions
sonata-project/doctrine-orm-admin-bundle 4.0.0-alpha-2 4.0.0-alpha-2 Integrate Doctrine ORM into the SonataAdminBundle
sonata-project/exporter                  2.6.2         2.6.2         Lightweight Exporter library
sonata-project/form-extensions           1.9.0         1.9.0         Symfony form extensions
sonata-project/twig-extensions           1.6.0         1.6.0         Sonata twig extensions
```

</p>
</details>

#### Symfony packages

<details><summary>show</summary>
<p>

```
symfony/asset                      v5.3.0  v5.3.0  Manages URL generation and versioning of web assets such as CSS stylesheets, JavaScript files and image files                                                                                                                                                            
symfony/browser-kit                v5.3.0  v5.3.0  Simulates the behavior of a web browser, allowing you to make requests, click on links and submit forms programmatically
symfony/cache                      v5.3.0  v5.3.0  Provides an extended PSR-6, PSR-16 (and tags) implementation                                                                                                                                                                                                                
symfony/cache-contracts            v2.4.0  v2.4.0  Generic abstractions related to caching                                                                    
symfony/config                     v5.3.0  v5.3.0  Helps you find, load, combine, autofill and validate configuration values of any kind                                                                                                                                                                                       symfony/console                    v5.3.0  v5.3.0  Eases the creation of beautiful and testable command line interfaces                                                                                                                                                                                                        
symfony/css-selector               v5.3.0  v5.3.0  Converts CSS selectors to XPath expressions                                                                                                                                                                                                                                 
symfony/debug-bundle               v5.3.0  v5.3.0  Provides a tight integration of the Symfony Debug component into the Symfony full-stack framework           
symfony/dependency-injection       v5.2.10 v5.3.0  Allows you to standardize and centralize the way objects are constructed in your application                
symfony/deprecation-contracts      v2.4.0  v2.4.0  A generic function and convention to trigger deprecation notices    
symfony/doctrine-bridge            v5.3.1  v5.3.1  Provides integration for Doctrine with various Symfony components                                                                                                                                                                                                           
symfony/dom-crawler                v5.3.0  v5.3.0  Eases DOM navigation for HTML and XML documents                                                                                                                                                                                                                             symfony/dotenv                     v5.3.0  v5.3.0  Registers environment variables from a .env file                                                        
symfony/error-handler              v5.3.0  v5.3.0  Provides tools to manage errors and ease debugging PHP code                                                                                                                                                                                                                 symfony/event-dispatcher           v5.3.0  v5.3.0  Provides tools that allow your application components to communicate with each other by dispatching events and listening to them
symfony/event-dispatcher-contracts v2.4.0  v2.4.0  Generic abstractions related to dispatching event                                                                                                                                                                                                                           symfony/expression-language        v5.3.0  v5.3.0  Provides an engine that can compile and evaluate expressions                                            
symfony/filesystem                 v5.3.0  v5.3.0  Provides basic utilities for the filesystem                                                                                                                                                                                                                                 
symfony/finder                     v5.3.0  v5.3.0  Finds files and directories via an intuitive fluent interface                                                                                                                                                                                                               
symfony/flex                       v1.13.3 v1.13.3 Composer plugin for Symfony                                                                                                                                                                                                                                                 symfony/form                       v5.3.0  v5.3.0  Allows to easily create, process and reuse HTML forms                                                       
symfony/framework-bundle           v5.2.10 v5.3.0  Provides a tight integration between Symfony components and the Symfony full-stack framework                
symfony/http-client                v5.3.0  v5.3.0  Provides powerful methods to fetch HTTP resources synchronously or asynchronously                           
symfony/http-client-contracts      v2.4.0  v2.4.0  Generic abstractions related to HTTP clients                                                                
symfony/http-foundation            v5.3.1  v5.3.1  Defines an object-oriented layer for the HTTP specification                                                 
symfony/http-kernel                v5.2.10 v5.3.1  Provides a structured process for converting a Request into a Response                                      
symfony/intl                       v5.3.0  v5.3.0  Provides a PHP replacement layer for the C intl extension that includes additional data from the ICU library 
symfony/ldap                       v5.3.0  v5.3.0  Provides a LDAP client for PHP on top of PHP's ldap extension                                               
symfony/lock                       v5.3.1  v5.3.1  Creates and manages locks, a mechanism to provide exclusive access to a shared resource                     
symfony/maker-bundle               v1.31.1 v1.31.1 Symfony Maker helps you create empty commands, controllers, form classes, tests and more so you can forget about writing boilerplate code.
symfony/monolog-bridge             v5.2.10 v5.3.0  Provides integration for Monolog with various Symfony components                                            
symfony/monolog-bundle             v3.7.0  v3.7.0  Symfony MonologBundle                                                                                       
symfony/options-resolver           v5.3.0  v5.3.0  Provides an improved replacement for the array_replace PHP function                                         
symfony/password-hasher            v5.3.0  v5.3.0  Provides password hashing utilities                                                                         
symfony/phpunit-bridge             v5.3.0  v5.3.0  Provides utilities for PHPUnit, especially user deprecation notices management                              
symfony/polyfill-php80             v1.23.0 v1.23.0 Symfony polyfill backporting some PHP 8.0+ features to lower PHP versions                                   
symfony/polyfill-php81             v1.23.0 v1.23.0 Symfony polyfill backporting some PHP 8.1+ features to lower PHP versions                                   
symfony/polyfill-uuid              v1.23.0 v1.23.0 Symfony polyfill for uuid functions                                                                         
symfony/process                    v5.3.0  v5.3.0  Executes commands in sub-processes                                                                          
symfony/property-access            v5.3.0  v5.3.0  Provides functions to read and write from/to an object or array using a simple string notation              
symfony/property-info              v5.3.1  v5.3.1  Extracts information about PHP class' properties using metadata of popular sources                          
symfony/psr-http-message-bridge    v2.1.0  v2.1.0  PSR HTTP message bridge                                                                                     
symfony/routing                    v5.3.0  v5.3.0  Maps an HTTP request to a set of configuration variables                                                    
symfony/security-acl               v3.1.2  v3.1.2  Symfony Security Component - ACL (Access Control List)                                                      
symfony/security-bundle            v5.2.10 v5.3.0  Provides a tight integration of the Security component into the Symfony full-stack framework                
symfony/security-core              v5.3.1  v5.3.1  Symfony Security Component - Core Library                                                                   
symfony/security-csrf              v5.3.0  v5.3.0  Symfony Security Component - CSRF Library                                                                   
symfony/security-guard             v5.2.10 v5.3.0  Symfony Security Component - Guard                                                                          
symfony/security-http              v5.2.10 v5.3.1  Symfony Security Component - HTTP Integration                                                               
symfony/semaphore                  v5.3.1  v5.3.1  Symfony Semaphore Component                                                                                 
symfony/serializer                 v5.3.1  v5.3.1  Handles serializing and deserializing data structures, including object graphs, into array structures or other formats like XML and JSON.
symfony/service-contracts          v2.4.0  v2.4.0  Generic abstractions related to writing services                                                            
symfony/sphinx                     v3.0.13 v3.0.13                             
symfony/stopwatch                  v5.3.0  v5.3.0  Provides a way to profile code                                                                              
symfony/string                     v5.3.0  v5.3.0  Provides an object-oriented API to strings and deals with bytes, UTF-8 code points and grapheme clusters in a unified way
symfony/swiftmailer-bundle         v3.5.2  v3.5.2  Symfony SwiftmailerBundle                                                                                   
symfony/templating                 v5.3.0  v5.3.0  Provides all the tools needed to build any kind of template system                                          
symfony/translation                v5.3.0  v5.3.0  Provides tools to internationalize your application                                                         
symfony/translation-contracts      v2.4.0  v2.4.0  Generic abstractions related to translation                                                                 
symfony/twig-bridge                v5.3.0  v5.3.0  Provides integration for Twig with various Symfony components                                               
symfony/twig-bundle                v5.2.10 v5.3.0  Provides a tight integration of Twig into the Symfony full-stack framework                                  
symfony/validator                  v5.3.1  v5.3.1  Provides tools to validate values                                                                           
symfony/var-dumper                 v5.3.0  v5.3.0  Provides mechanisms for walking through any arbitrary PHP variable                                          
symfony/var-exporter               v5.3.0  v5.3.0  Allows exporting any serializable PHP data structure to plain PHP code                                      
symfony/web-link                   v5.2.10 v5.3.0  Manages links between resources                                                                             
symfony/web-profiler-bundle        v5.2.10 v5.3.0  Provides a development tool that gives detailed information about the execution of any request              
symfony/webpack-encore-bundle      v1.11.2 v1.11.2 Integration with your Symfony app & Webpack Encore!                                                         
symfony/yaml                       v5.3.0  v5.3.0  Loads and dumps YAML files       
```

</p>
</details>

#### PHP version

```
PHP 7.4.19 (cli) (built: May 12 2021 13:21:40) ( NTS )
Copyright (c) The PHP Group
Zend Engine v3.4.0, Copyright (c) Zend Technologies
    with Zend OPcache v7.4.19, Copyright (c), by Zend Technologies
    with blackfire v1.51.0~linux-x64-non_zts74, https://blackfire.io, by Blackfire
```

## Subject

When hosting assets files on a dedicated CDN under a dedicated domain/path, the Fonts (used for icons) are not loaded

The path to the font files is currently hard coded in `app.css`, and expect the font to be located in `/bundles/sonataadmin/fonts/...` which is not always the case.


## Steps to reproduce

- Upload Sonata's `Resources/public` folder in a s3 sub-directory . ie: `assets.myapp.com/version1`
- Configure the App to use S3 
```
framework:
    assets:
        base_urls:
            - 'http://assets.myapp.com/version1/'
```

app.css is downloaded, but fonts files throws 404, because tried to be downloaded from `http://assets.myapp.com/bundle/sonata...` instead of `http://assets.myapp.com/version1/bundle/sonata`

Maybe we should use relative path: `./fonts/file.ext` instead of `/bundles/sonataadmin/fonts/file.ext`

**Comment by VincentLanglet** [comment]
Do you have a solution for this @jordisala1991 ?

**Comment by jordisala1991** [comment]
Maybe it needs some modifications in the webpack file, I dont know really. The thing is we dont print any url by hand, so is all handled by webpack on master

**Comment by jderusse** [comment]
Should be fixed by #7253
